src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -117,15 +123,40 @@ export const Browser = () => {
   );
 };
 
-const WasmLoadingBrowserContainer = () => {
-  useEffect(() => {
-    /* eslint-disable */
-    // @ts-ignore ensembl-genome-browser does not have typescript definitions
-    import('ensembl-genome-browser');
-    /* eslint-enable */
-  });
+type GenomeBrowserContextType = {
+  genomeBrowser: EnsemblGenomeBrowser;
+  setGenomeBrowser: (genomeBrowser: EnsemblGenomeBrowser) => void;
+  zmenus: StateZmenu;
+  setZmenus: (zmenus: StateZmenu) => void;
+};
 
-  return <Browser />;
+export const GenomeBrowserContext = React.createContext<
+  GenomeBrowserContextType | undefined
+>(undefined);
+
+const GenomeBrowserInitContainer = () => {
+  const [genomeBrowser, setGenomeBrowser] = useState<EnsemblGenomeBrowser>();
+
+  const [zmenus, setZmenus] = useState<StateZmenu>({});
+
+  const browser = useMemo(() => {
+    return <Browser />;
+  }, []);
+
+  return (
+    <GenomeBrowserContext.Provider
+      value={
+        {
+          genomeBrowser,
+          setGenomeBrowser,
+          zmenus,
+          setZmenus
+        } as GenomeBrowserContextType
+      }
+    >
+      {browser}
+    </GenomeBrowserContext.Provider>
+  );
 };
 
-export default WasmLoadingBrowserContainer;
+export default memo(GenomeBrowserInitContainer, isEqual);
```

Comment:
`GenomeBrowserInitContainer` isn't receiving any props from the outside; so the `isEqual` function should be unnecessary.